patches/server/0009-Timings-v2.patch

```diff
@@ -1222,24 +1216,29 @@ index ef336645f3bd7a86129ad1dff7e1a15dc93d1e3e..d0b0fdaf5451bcc7f7ac7dab28aa59ef
              this.chunkMap.getChunks().forEach((playerchunk) -> { // Paper - no... just no...
                  Optional<LevelChunk> optional = ((Either) playerchunk.getTickingChunkFuture().getNow(ChunkHolder.UNLOADED_LEVEL_CHUNK)).left();
  
-                 if (optional.isPresent()) {
+@@ -631,6 +640,13 @@ public class ServerChunkCache extends ChunkSource {
                      this.level.getProfiler().push("broadcast");
+                     LevelChunk chunk = (LevelChunk) optional.get();
+ 
 +                    this.level.timings.broadcastChunkUpdates.startTiming(); // Paper - timings
-                     playerchunk.broadcastChanges((LevelChunk) optional.get());
++                    playerchunk.broadcastChanges((LevelChunk) optional.get());
 +                    this.level.timings.broadcastChunkUpdates.stopTiming(); // Paper - timings
++
++                    this.level.getProfiler().pop();
++                    Optional<LevelChunk> optional1 = ((Either) playerchunk.getEntityTickingChunkFuture().getNow(ChunkHolder.UNLOADED_LEVEL_CHUNK)).left();
++
+                     playerchunk.broadcastChanges(chunk);
```

Comment:
This is wrong: you're broadcasting changes twice. Simply surround the bottom broadcastChanges with the timings updates